lib/models/bundle.js

```diff
@@ -234,9 +44,9 @@ Bundle.prototype.validate = function () {
 
   codes.forEach(function (code) {
     // Single number refs -> real code.
-    if (code.ref !== null) {
-      var singleRef = codes[code.ref];
-      if (!singleRef.isCode) {
+    if (code.isSingleRef()) {
+      var singleRef = _.find({ id: code.singleRef })(codes);
```

Comment:
Sure! Check out https://github.com/FormidableLabs/inspectpack/blob/4909a7644cf29a8f0c3df8498d10b4125eba2467/test/fixtures/basic-lodash-object-expression.js

Definitely the first time I've run into this...